packages/mask/src/components/DataSource/useNextID.ts

```diff
@@ -31,11 +31,11 @@ const verifyPersona = (personaIdentifier?: PersonaIdentifier, username?: string)
     })
 }
 
-export const useNextIDBoundByPlatform = (platform?: NextIDPlatform, identity?: string) => {
-    const res = useAsyncRetry(() => {
-        if (!platform || !identity) return Promise.resolve([])
-        return NextIDProof.queryExistedBindingByPlatform(platform, identity)
-    }, [platform, identity])
+export const useNextIDBoundByPlatform = (platform?: NextIDPlatform, userId?: string) => {
```

Comment:
`identity` may be a more appropriate term, since it could be address of a wallet.